include/libtorrent/settings_pack.hpp

```diff
@@ -1093,7 +1103,9 @@ namespace libtorrent
 
 			// this is the limit on the time a torrent has been an active seed
 			// (specified in seconds) before it is considered having met the seed
-			// limit criteria. See queuing_.
+			// limit criteria. See queuing_. It does not disable the torrent, it
+			// does only move to a lower queue position which may result into an
```

Comment:
this sentence is a bit odd. Perhaps something like: "When this limit is hit, the torrent is not disabled, instead its seed rank is adjusted give priority to torrents that have not met their seed_time_limit"